src/fonts/gonville_glyphs.ts

```diff
@@ -1,876 +1,1087 @@
+      x_min: 0,
```

Comment:
What happens if we don't round these values? Does the visual output change? Is there any benefit to preserving one or two (or more) decimal points?

Reply:
See above.